styleguide/components/21-atoms/buttons/button-alert/button-alert.twig

```diff
@@ -1,7 +1,12 @@
 <button
   type="button"
-  class="button button-alert {{ modifier }}"
+  class="button button-alert {{ modifier }} {% if icon %} button-icon {% endif %}"
```

Comment:
Discuss with design and UX if these icons are indeed optional.

There's no WCAG guideline directly applicable, but consider a combination of the following:

* WCAG 3.2.4 Consistent Identification: Components that have the same functionality within a set of Web pages are identified consistently. (Level AA) 
* WCAG 1.4.1 Use of Color: Color is not used as the only visual means of conveying information, indicating an action, prompting a response, or distinguishing a visual element. (Level A) 

So all alert buttons should have a consistent look and this cannot be done by use of colour alone.
Same goes for download buttons, link buttons...

Reply:
Agreed, this should be discusses with design.